conftest.py

```diff
@@ -218,7 +218,7 @@ def pytest_collection_modifyitems(session, config, items):
             for forced in config.option.force_execute:
                 if item.nodeid.startswith(forced):
                     logger.info(f"{item.nodeid} is normally skipped, but forced thanks to -F {forced}")
-                    item.own_markers = [m for m in item.own_markers if m.name not in ("skip", "skipif")]
+                    item.own_markers = [m for m in item.own_markers if m.name not in ("skip", "skipif", "xfail")]
```

Comment:
This will still skip tests that we want to force. Take for instance the test `tests/appsec/test_traces.py::Test_CollectDefaultRequestHeader`
It is currently set as a missing_feature in the ruby.yml manifest. But the actual test is `tests/appsec/test_traces.py::Test_CollectDefaultRequestHeader::test_collect_default_request_headers`, with item.parent set to `tests/appsec/test_traces.py::Test_CollectDefaultRequestHeader`
So when you look at its own_markers, it is empty, but when you iterate on them using iter_markers, you can still see the missing_feature marker.
That is because it also looks at the parent's own_markers, which in this case actually contains the missing_feature marker, and it most likely also looks at the entire ancestry